testDistro.sh

```diff
@@ -1,17 +1,34 @@
 #!/bin/bash
-set -e #stop on error
-# set -x #verbose on
 
+set -e # Stop on error
+set -u # Treat unset variables as an error
+# set -x # Uncomment this for debugging
+
+# Define log file path
+LOG_FILE="/var/log/my_script.log"
```

Comment:
```suggestion
LOG_FILE="target/testDistro.log"
```

Reply:
context: 
1) the file name you suggested is very generic, 
2) users can't typically write there:
```
tee: /var/log/my_script.log: Permission denied
```

Reply:
you marked this as resolved but forgot to make the change... would you mind doing that so we can get this merged? thanks!